config/jetpack-cloud-development.json

```diff
@@ -83,5 +83,5 @@
 	"restricted_me_access": false,
 	"theme_color": "#2fb41f",
 	"google_analytics_key": "UA-52447-43",
-	"hotjar_enabled": false
+	"hotjar_enabled": true
```

Comment:
HotJar is previously already enabled in all Calypso blue environments (wordpress.com), So for consistency, I'll enable HotJar in all Calypso green environments (Jetpack cloud) too.